src/Common/src/System/CodeDom/CodeObject.cs

```diff
@@ -5,7 +5,11 @@
 using System.Collections;
 using System.Collections.Specialized;
 
+#if !Feature_Serialization
 namespace System.CodeDom
+#else
+namespace System.Runtime.Serialization
+#endif
 {
     [Serializable]
     public class CodeObject
```

Comment:
Please also condition the visibility. We want to avoid having these be public in the serialization library even if they are in a different namespace.